### PR TITLE
Format examples and add another example

### PR DESCRIPTION
**Pull request by EmpathyYT**
## Summary

Show cases how to extract user info

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, ...)

### CODE REVIEW

File: examples/Info.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+bot = commands.Bot(command_prefix="?", description=description, intents=intents)
+
+
+@bot.event
+async def on_ready():
+    print(f"Logged in as {bot.user} (ID: {bot.user.id})")
+    print("------")
+
+
+ @bot.command()
```
Comment (MhmCats) [inline examples/Info.py:20]:
```suggestion
@bot.command()
```

File: examples/Info.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        if colour.value:
+            e.colour = colour
+
+        if user.avatar:
+            e.set_thumbnail(url=user.avatar_url)
+
+        if isinstance(user, discord.User):
+            e.set_footer(text='This member is not in this server.')
+
+        await ctx.send(embed=e)
```
Comment (MhmCats) [inline examples/Info.py:55]:
```suggestion
async def info(ctx, *, user: Union[discord.Member, discord.User] = None):
    """Shows info about a user."""

    user = user or ctx.author
    e = discord.Embed()
    roles = [role.name.replace('@', '@\u200b') for role in getattr(user, 'roles', [])]
    e.set_author(name=str(user))



    e.add_field(name='ID', value=user.id, inline=False)
    e.add_field(name='Joined', value=round_time(user.joined_at), inline=False)
    e.add_field(name='Created', value=round_time(user.created_at), inline=False)

    voice = getattr(user, 'voice', None)
    if voice is not None:
        vc = voice.channel
        other_people = len(vc.members) - 1
        voice = f'{vc.name} with {other_people} others' if other_people else f'{vc.name} by themselves'
        e.add_field(name='Voice', value=voice, inline=False)

    if roles:
        e.add_field(name='Roles', value=', '.join(roles) if len(roles) < 10 else f'{len(roles)} roles', inline=False)

    colour = user.colour
    if colour.value:
        e.colour = colour

    if user.avatar:
        e.set_thumbnail(url=user.display_avatar.url)

    if isinstance(user, discord.User):
        e.set_footer(text='This member is not in this server.')

    await ctx.send(embed=e)```

File: examples/Info.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        user = user or ctx.author
+        e = discord.Embed()
+        roles = [role.name.replace('@', '@\u200b') for role in getattr(user, 'roles', [])]
+        e.set_author(name=str(user))
+
+        
+
+        e.add_field(name='ID', value=user.id, inline=False)
+        e.add_field(name='Joined', value=round_time(user.joined_at), inline=False)
+        e.add_field(name='Created', value=round_time(user.created_at), inline=False)
```
Comment (MhmCats) [inline examples/Info.py:33]: Where is `round_time` defined?

**Review by MhmCats — CHANGES_REQUESTED** [review]
Also this is not really consistent with what quotation marks you use for string literals.

File: examples/Info.py (hunk 3 above)
Comment (EmpathyYT) [reply examples/Info.py]: Gonna fix soon

**Comment by EmpathyYT** [comment]
This example is more newbie-friendly than the old one

**Comment by MhmCats** [comment]
If this is a slash command example then shouldn't it go in `examples/app_commands`?

**Comment by EmpathyYT** [comment]
it's not a slash

**Review by nerdguyahmad — CHANGES_REQUESTED** [review]

**Comment by nerdguyahmad** [comment]
> it's not a slash

Why're you using discord.Bot then/

**Comment by EmpathyYT** [comment]
> > it's not a slash
> 
> Why're you using discord.Bot then/

mb then

File: examples/Info.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+async def userinfo(ctx, user: discord.Member):
+    user = user or ctx.author
+    e = discord.Embed()
+    e.set_author(name=user.name)
+    e.add_field(name='ID', value=user.id, inline=False)
+    e.add_field(name='Joined', value=user.joined_at.strftime(
+        "%A, %B %d %Y @ %H:%M:%S %p"), inline=False)
+    e.add_field(name='Created', value=user.created_at.strftime(
+        "%A, %B %d %Y @ %H:%M:%S %p"), inline=False)
+    await ctx.send(embed=e, colour=user.colour)
```
Comment (nerdguyahmad) [inline examples/Info.py:33]: The color should be in `Embed()`, I made a typo in the review suggestion

**Review by nerdguyahmad — CHANGES_REQUESTED** [review]

File: examples/Info.py (hunk 4 above)
Comment (EmpathyYT) [reply examples/Info.py:33]: yeah i was like since when that exists

**Comment by Lulalaby** [comment]
@EmpathyYT please comment your code

**Comment by Dorukyum** [comment]
Please check the style too (file name, indentation, empty line count etc.)

File: examples/app_commands/Info.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+from discord.ext import commands
+# imports
+
+intents = discord.Intents(
+            guilds=True,
+            members=True,
+            messages=True,
+            )
+
+bot = commands.Bot(command_prefix=".", description="An example to showcase how to extract info about users",intents=intents)
```
Comment (RPSMain) [inline examples/app_commands/Info.py:11]:
```suggestion
bot = commands.Bot(command_prefix="/", description="An example to showcase how to extract info about users",intents=intents)
```

**Review by RPSMain — APPROVED** [review]

File: examples/app_commands/Info.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    colour = user.colour
+    if colour.value: # if user has a role with a color
+           e.colour = colour
+    
+    if isinstance(user, discord.User): # checks if the user in the server
+        e.set_footer(text='This member is not in this server.')
+
+    await ctx.respond(embed=e) # sends the embed
+
+bot.run("urtoken")
```
Comment (krittick) [inline examples/app_commands/Info.py:32]:
```suggestion
bot.run("your token")
```

More professional wording.

File: examples/app_commands/Info.py [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+            )
+
+bot = commands.Bot(command_prefix=".", description="An example to showcase how to extract info about users",intents=intents)
+
+@bot.slash_command(name="userinfo", description="gets the info of a user")
+async def info(ctx, user: discord.Member = None):
+    user = user or ctx.author #if no user is provided it'll use the the author of the message
+    e = discord.Embed()
+    e.set_author(name=user.name)
+    e.add_field(name='ID', value=user.id, inline=False) # user ID
```
Comment (krittick) [inline examples/app_commands/Info.py:18]:
```suggestion
    e.add_field(name="ID", value=user.id, inline=False) # user ID
```
Keeps consistency with the double quoted strings the file sets precedent with.

File: examples/app_commands/Info.py [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    e.add_field(name='Joined',
+                value=f"<t:{round(user.joined_at.timestamp())}:F>", inline=False) # When the user joined the server
```
Comment (krittick) [inline examples/app_commands/Info.py:20]:
```suggestion
    e.add_field(name="Joined", value=discord.utils.format_dt(round(user.joined_at.timestamp()), "F"), inline=False) # When the user joined the server
```
Switch to using `discord.utils.format_dt` for better readability.

File: examples/app_commands/Info.py [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    e.add_field(name='Created',
+                value=f"<t:{round(user.created_at.timestamp())}:F>", inline=False) # When the user's account was created
```
Comment (krittick) [inline examples/app_commands/Info.py:22]:
```suggestion
    e.add_field(name="Created", value=discord.utils.format_dt(round(user.created_at.timestamp()), "F"), inline=False) # When the user's account was created
```
Switch to using `discord.utils.format_dt` for better readability.

**Comment by krittick** [comment]
“Info.py” should be renamed to “info.py” per [PEP8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)

File: examples/app_commands/Info.py [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+    e.set_author(name=user.name)
+    e.add_field(name="ID", value=user.id, inline=False) # user ID
+    e.add_field(name="Joined", value=discord.utils.format_dt(round(user.joined_at.timestamp()), "F"), inline=False) # When the user joined the server
+    e.add_field(name="Created", value=discord.utils.format_dt(round(user.created_at.timestamp()), "F"), inline=False) # When the user's account was created
+    colour = user.colour
+    if colour.value: # if user has a role with a color
+           e.colour = colour
+    
+    if isinstance(user, discord.User): # checks if the user in the server
+        e.set_footer(text='This member is not in this server.')
```
Comment (krittick) [inline examples/app_commands/Info.py:26]:
```suggestion
        e.set_footer(text="This member is not in this server.")
```
I missed another one - update to double quotes to match the rest.

**Pull request merged**